### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by Stradivario** [Bunlong/local-async-storage#1]
Hello there! I am just passing here to ask some questions

1. Why do you think that wrapping LocalStorage with Promise will give you a better performance ? 
2. Do you know that wrapping Sync function with Async doesn't really give you asyncronious behaviour ? 
3. Have you consider using this library ? https://github.com/localForage/localForage i think you will love it since it is wrapper for LocalStorage, IndexedDb or WebSQL.

Nothing in personal i just check your code and i was curious if you think about these scenarios!

Best Regards,
Kristiyan Tachev

**Comment by vzakharov** [comment]
Yeah, looking at the code, it just defers the execution until the call stack is empty, but it will still block execution afterwards.
